### ISSUE TITLE
Batching log input

### ISSUE THREAD

**Issue by darakian** [mozilla/MozDef#1246]
Hey all,

Is it possible to batch log input into MozDef? Looking at the docs here
https://mozdef.readthedocs.io/en/latest/usage.html
I don't see any mention of it. I've tried bundling my logs as my desired format of a json array of messages
`[{msg1}, {msg2}, ...]` and the input gets rejected.
In the code I see 
```
@route('/_bulk',method='POST')
@route('/_bulk/',method='POST')
def bulkindex():
```
and sending logs to `mozdef:8080/_bulk` gives back a 200, but I don't see my logs appearing. Is this an under construction section of the code or am I misreading things?

Edit:
Data format sent to `_bulk`
`'[{"summary": "just a double test"}, {"summary": "just a double test"}]'`

**Comment by Phrozyn** [comment]
The loginput code may require each event on a single line rather than separated by a comma. 

The first thing the script does is add each event line by line to an "eventlist" list.

Otherwise it may hit a character limit by attempting to index every event as a single event. 

You should have a log output called loginput.log wherever you've directed mozdef to write logs. 

Do you see anything in the log?

**Comment by darakian** [comment]
Ah I see. I'll test with newlines and check my logs on monday. For my log output it may be easier to send comma-space separated values. Would you be opposed to enabling general white space separated arrays or perhaps a configurable option? I don't mind making the PR for this if so.

**Comment by darakian** [comment]
PR here
https://github.com/mozilla/MozDef/pull/1247

**Comment by Phrozyn** [comment]
Merged #1247
Closing
